src/frontend/src/pages/index.js

```diff
@@ -92,16 +92,14 @@ export default function IndexPage() {
   }, [currentNumPosts]);
 
   return (
-    <PageBase>
-      <>
-        <Banner className="banner" />
-        <ScrollToTop />
-        <main className="main">
-          {posts.length > 0 ? <Posts posts={posts} /> : null}
-          {currentNumPosts !== initNumPosts ? <CustomizedSnackBar posts={currentNumPosts} /> : null}
-        </main>
-        <footer>© {new Date().getFullYear()}</footer>
-      </>
+    <PageBase title="Home">
+      <Banner className="banner" />
+      <ScrollToTop />
+      <main className="main">
+        {posts.length > 0 ? <Posts posts={posts} /> : null}
```

Comment:
In a follow-up, we should move this check into `<Posts>` instead.

Reply:
I'm not 100% how to describe this in an issue. Maybe @c3ho is a better candidate to file the issue since he wrote this piece of the code.

Reply:
@humphd I made an attempt to move the check into `<Posts>`, can you review this PR again? Thanks